sdk/webpubsub-socketio-extension/examples/chat-with-negotiate/index.js

```diff
@@ -25,23 +25,28 @@ const wpsOptions = {
 };
 */
 
-// Add an Web PubSub Option
+function authentiacte(username) {
+    if (username.length < 10) return true;
```

Comment:
Is this tricky? Can we just return true but add comments about "Add your Auth logic here"

Reply:
updated as suggested